apps/server/src/modules/user/user.module.ts

```diff
@@ -7,14 +7,29 @@ import { RoleModule } from '@src/modules/role/role.module';
 import { RoleService } from '@src/modules/role/service/role.service';
 import { RoleUc } from '@src/modules/role/uc/role.uc';
 import { SchoolModule } from '@src/modules/school/school.module';
+import { TransactionUtil } from '@shared/common/utils/transaction.util';
+import { AccountModule } from '@src/modules/account';
+import { AccountRepo } from '@src/modules/account/repo/account.repo';
 import { UserController } from './controller';
 import { UserService } from './service/user.service';
 import { UserUc } from './uc';
 
 @Module({
-	imports: [SchoolModule, RoleModule, LoggerModule],
+	imports: [SchoolModule, RoleModule, AccountModule, LoggerModule],
 	controllers: [UserController],
-	providers: [UserRepo, UserDORepo, PermissionService, UserUc, UserService, RoleRepo, RoleUc, SchoolRepo, RoleService],
+	providers: [
+		UserRepo,
+		UserDORepo,
+		PermissionService,
+		UserUc,
+		UserService,
+		RoleRepo,
+		RoleUc,
+		SchoolRepo,
+		RoleService,
+		AccountRepo,
```

Comment:
please do not provide the account repo here. instead use a service provided by the accountModule

Reply:
done